vite.config.js

```diff
@@ -10,9 +11,15 @@ export default defineConfig({
     }),
     checker({
       typescript: true
-    })
+    }),
+    nodePolyfills(),
   ],
   build: {
+    minify: false,
+    terserOptions: {
+      compress: false,
+      mangle: false,
+    },
```

Comment:
We may want to revert this, but it makes error investigations a lot easier for now